scripts/cleanup.sh

```diff
@@ -5,6 +5,7 @@ set -e
 sed -i '' -e 's/^PermitRootLogin yes/#PermitRootLogin no/' /etc/ssh/sshd_config
 
 # Purge files we no longer need
+set +e
```

Comment:
I'm curious, what prompted you to set it?

Reply:
rm -f can return nonzero error codes, despite the man page claiming otherwise. On mine rm -f was erroring out ($? = 1) for the directory that did not exist. Silently, but the error code killed it.

I thought that removing the failing lines would have caused more harm in breaking backward compatibility, the original reason for the delete, than ignoring error codes which might indicate something wrong with the file system at this point in the script.

Reply:
That sounds like an `rm` bug. I'll try to replicate it.

Reply:
The fix landed as https://github.com/freebsd/freebsd-src/commit/0b2c159c8faa3ca155412a1b2e57b3d6bcf91880. Thank you for reporting it.